addons/web_editor/models/field_html_history_mixin.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+from .patch_utils import apply_patch, generate_comparison, generate_patch
+
+
+class HtmlHistory(models.AbstractModel):
+    _name = "field.html.history.mixin"
+    _description = "Field html History"
+    _max_history_revisions = 300
+
+    history_revisions = fields.Json("History revisions Data")
```

Comment:
Shouldn't we prefetch False ?

Reply:
> oh, and the name seems a little bit collision-prone, no? don't we usually have stronger namespace for mixins?
 

would `html_field_history_revisions`  be better ?